openprescribing/media/js/src/measure_utils.js

```diff
@@ -67,11 +67,20 @@ var utils = {
         panelData = _this._rollUpByOrg(panelData[0], options.orgType);
       }
       panelData = _this._getSavingAndPercentilePerItem(panelData,
-        numMonths);
-      data = _.sortBy(panelData, function(d) {
-        if (d.meanPercentile === null) return -1;
-        return d.meanPercentile;
-      }).reverse();
+                                                       numMonths);
+      if (options.rollUpBy == 'measure_id' || options.lowIsGood) {
```

Comment:
I think this is the issue. `panelData` can be structured such that each chart panel is a measure (on pages like `/ccg/XXX/measures`) or such that each chart panel is an organisation (on pages like `/ccg/XXX/statins`). We'll need to amend the sorting code for each of these scenarios.